dependency-versions.gradle

```diff
@@ -47,6 +47,7 @@ dependencyManagement {
       entry 'logl-logl'
     }
     dependency('org.mapdb:mapdb:3.0.7')
+    dependency('org.miracl.milagro.amcl:milagro-crypto-java:0.3.0')
```

Comment:
hey, 0.4.0 is out: https://mvnrepository.com/artifact/org.miracl.milagro.amcl/milagro-crypto-java/0.4.0

Worth upgrading?

Reply:
Ya good idea